docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md

```diff
@@ -177,7 +177,29 @@ vbc Example1.vb
 ```  
 Bon jour!  
 ```  
+
+### .NET Core resource fallback process
+ The .NET Core resource fallback process involves the following steps:  
   
+1.  The runtime checks the directory of the currently executing assembly for a directory that matches the requested culture. If it finds the directory, it searches that directory for a valid satellite assembly for the requested culture. The runtime then searches the satellite assembly for the requested resource. If it finds the resource in the assembly, it uses it. If it doesn't find the resource, it continues the search.  
+  
+2.  The runtime raises the <xref:System.Runtime.Loader.AssemblyLoadContext.Resolving?displayProperty=nameWithType> event to indicate that it is unable to find the satellite assembly. If you choose to handle the event, your event handler can return a reference to the satellite assembly whose resources will be used for the lookup. Otherwise, the event handler returns `null` and the search continues.  
+  
+3. The runtime next searches the parent culture assemblies through many potential levels, each time repeating steps 1 & 2. 
+
+The parent culture is defined as the appropriate fallback culture. Consider parents as fallback candidates, because providing any resource is preferable to throwing an exception. This process also allows you to reuse resources. You should include a particular resource at the parent level only if the child culture doesn't need to localize the requested resource. For example, if you supply satellite assemblies for en (neutral English), en-GB (English as spoken in the United Kingdom), and en-US (English as spoken in the United States), the en satellite would contain the common terminology, and the en-GB and en-US satellites could provide overrides for only those terms that differ.  
```

Comment:
- This needs to be indented two spaces to continue to numbered list item. 
- the appropriate fallback culture --> an appropriate fallback culture.